clang/include/clang/AST/Type.h

```diff
@@ -5456,9 +5459,8 @@ class DeducedTemplateSpecializationType : public DeducedType,
 /// TemplateArguments, followed by a QualType representing the
 /// non-canonical aliased type when the template is a type alias
 /// template.
-class alignas(8) TemplateSpecializationType
-    : public Type,
-      public llvm::FoldingSetNode {
+class alignas(8) TemplateSpecializationType : public Type,
```

Comment:
Unrelated change?  Or am I missing something?

Reply:
This was a clang-format error because of a merge conflict. Reverted it.